packages/Card/Card.jsx

```diff
@@ -7,6 +7,8 @@ import safeRest from '../../shared/utils/safeRest'
 import styles from './Card.modules.scss'
 
 /**
+ * @version 0.0.0-development
```

Comment:
Does this work right? I was thinking the "@version" had to come AFTER the description?

Reply:
It works fine. I put it before the description because not all components have a description. 
I like the consistency of having the version number right after the import statement.